core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -788,6 +788,8 @@ private void handleLocaleCheck() {
     tableDrawerAdapter.setTableClickListener(null);
     tableDrawerAdapter = null;
     unbinder.unbind();
+    toolbar = null;
```

Comment:
did you discover a new leak?

Reply:
Hmm, don't remember why I put that there. It used to cause issues, setting `toolbar=null`, will investigate.

Reply:
This does not cause any issues and I think this is the most memory-leak free branch I've tried so far, so that's a good sign!

Reply:
But is it pointless code though? Just adding code cause "it might do something" is a way to a big bowl of rigatoni

Reply:
So very true, thanks for reminding me.